pandas/core/series.py

```diff
@@ -435,6 +435,51 @@ def dtypes(self) -> DtypeObj:
 
     @property
     def name(self) -> Label:
+        """
+        Return the name of the Series.
+
+        The name of a Series becomes its index or column name if it is used
+        to form a DataFrame. It is also used whenever displaying the Series
+        using the interpreter, and persists through some transformations.
+
+        Returns
+        -------
+        Label (int, str or other hashable object).
+
+        See Also
+        --------
+        Index.name : Equivalent property for Index object.
+
+        Examples
+        --------
+        >>> s1 = pd.Series([1, 2, 3], dtype=np.int64)
+        >>> s1
+        0    1
+        1    2
+        2    3
+        dtype: int64
+        >>> s1.name = "Numbers"
```

Comment:
Can you post a screenshot of the built doc? *Might* need a newline here

Reply:
![2020-03-14 (1)](https://user-images.githubusercontent.com/42591328/76686071-1160e600-6619-11ea-844c-5de89fe7ab47.png)

Though I changed this here's a screenshot of the new version